src/rhsmlib/facts/virt.py

```diff
@@ -34,7 +32,8 @@ def get_virt_info(self):
         virt_dict = {}
 
         try:
-            host_type = compat_check_output('/usr/sbin/virt-what')
+            host_type_raw: bytes = subprocess.check_output('/usr/sbin/virt-what')
+            host_type: str = host_type_raw.decode("utf-8").strip()
```

Comment:
i think there is no need for `strip()` here, as it was not done before

Reply:
You are right; the `striplines()` below it can deal with trailing newline.